.github/workflows/lint.yaml

```diff
@@ -1,5 +1,5 @@
 on: push
-name: Test
+name: Lint
```

Comment:
"Lint" is probably a bit less misleading as we're not really testing anything here.